src/types/contracts/derivativeScrt/model.ts

```diff
@@ -0,0 +1,20 @@
+import { StakingDerivativesValidator } from './response';
+
+type StakingDerivativesInfo = {
+  validators: StakingDerivativesValidator[],
+  supply: number,
+  exchangeRate: number,
+  communityRewards: number,
+  nextUnboundAmount: number,
+  nextUnbondingBatchEstimatedTime: number,
+}
+
+type StakingDerivativesFee = {
```

Comment:
It's fine to individually parse this query, but I think a batch query makes more sense for almost all use cases. There's no reason I can think of we wouldn't always want fees returned at the same time as the rest of the staking data. Let's create a single data model with both normal info and fees and return it as a batch query.